lib/hexpm_web/controllers/package_report_controller.ex

```diff
@@ -190,13 +209,22 @@ defmodule HexpmWeb.PackageReportController do
 
   defp build_report_form(conn, params) do
     %{"repository" => repository, "package" => name} = params
+
+    description =
+      if params["description"] == nil do
+        " "
+      else
+        params["description"]
+      end
```

Comment:
It is better to have this check in the template. Usually `nil` is rendered as an empty string, so make sure we even need the check.